### DISCUSSION TITLE
webpack4+nuxt+nest(webpack4)+fastift+any orm = not work

### DISCUSSION THREAD

**Discussion by n4an** [nuxt/nuxt#9841] [Questions (Nuxt 2)]
If to main.ts from this example to add any database orm (sequalize/prisma/typeorm)

Prism controller from https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nestjs
import {
  Controller,
  Get,
  Param
  // Post,
  // Body,
  // Put,
  // Delete,
  // Query
} from '@nestjs/common'
import {
  // User as UserModel,
  Post as PostModel
  // Prisma
} from '@prisma/client'
import { AppService } from './app.service'
@Controller()
export class AppController {
  // eslint-disable-next-line no-useless-constructor
    constructor (private readonly prismaService: AppService) {}

    @Get('post/:id')
    async getPostById (@Param('id') id: string): Promise<PostModel> {
      return await this.prismaService.post.findUnique({ where: { id: Number(id) } })
    }
get an error:

ERROR [ExceptionsHandler] Cannot read property 'post' of undefined TypeError: Cannot read property 'post' of undefined

Sequalize controller from https://github.com/nestjs/nest/tree/master/sample
    import { Body, Controller, Delete, Get, Param, Post } from '@nestjs/common'
    import { CreateUserDto } from './dto/create-user.dto'
    import { User } from './user.model'
    import { UsersService } from './users.service'

    @Controller('users')
    export class UsersController {
      // eslint-disable-next-line no-useless-constructor
      constructor (private readonly usersService: UsersService) {}

      @Post()
      create (@Body() createUserDto: CreateUserDto): Promise<User> {
        return this.usersService.create(createUserDto)
      }
ERROR [ExceptionsHandler] Cannot read property 'findOne' of undefined TypeError: Cannot read property 'findOne' of undefined

Main.ts from

  import { NestFactory } from '@nestjs/core';
  import { FastifyAdapter, NestFastifyApplication } from '@nestjs/platform-fastify'; // npm i --save @nestjs/platform-fastify
  import { AppModule } from './app.module';

  async function bootstrap() {
    const app = await NestFactory.create<NestFastifyApplication>(AppModule, new FastifyAdapter({ ignoreTrailingSlash: true, logger: true }));
    await app.init();

    const fastify = app.getHttpAdapter().getInstance();
    await fastify.ready();

    // Unlike the Express version which the app instance itself is a handler function, the fastify instance is an object
    // we have to return a handler function for Nuxt serverMiddleware
    return function (req, res) {
      fastify.server.emit('request', req, res);
    };
  }

  export default bootstrap;

**Comment by danielroe** [comment]
@Mitch-i Would you provide a reproduction? :pray: What makes you think this is a Nuxt issue, given that the code that gives the error is from your nest code?

**Comment by n4an** [comment]
the repro

> @Mitch-i Would you provide a reproduction?  What makes you think this is a Nuxt issue, given that the code that gives the error is from your nest code?

reproduction example https://github.com/Mitch-i/nuxt_nest

I don't think that is nuxt error, I need some help and wrote to all
